Tests/PrivateTest.cs

```diff
@@ -65,6 +65,29 @@ public void TestCacheableExposePrivateMethodViaType()
             var tCachedInvoke = new CacheableInvocation(InvocationKind.InvokeMember, "Test", context: typeof(TestWithPrivateMethod));
             Assert.That( tCachedInvoke.Invoke(tTest), Is.EqualTo(3));
         }
+
+        [TestCase(typeof(TestNestedWithPrivateStaticField))]
+        [TestCase(typeof(TestNonPublicWithPrivateStaticField))]
+        [TestCase(typeof(TestWithPrivateStaticField))]
+        public void TestPrivateStaticField(Type type)
+        {
+            var staticContext = InvokeContext.CreateStatic;
+            try
+            {
+                Dynamic.InvokeSet(staticContext(type), "Hello", null);
+            }
+            catch (RuntimeBinderException)
+            {
+
+            }
```

Comment:
Removing this `try..catch` causes all test cases to fail but I do not understand why